soundbay/data.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, data_path, metadata_path, augmentations, augmentations_p, pre
         self.audio_dict = self._create_audio_dict(Path(data_path))
         self.metadata_path = metadata_path
         metadata = pd.read_csv(self.metadata_path)
+        metadata_filenames = pd.read_csv(self.metadata_path, usecols=['filename'], dtype=pd.StringDtype())
```

Comment:
what  `metadata_filenames` is used for?

Reply:
would that mean to dictate the type of all the possible columns in the metadata (which could be different with different datasets) or can I define a dict such as `dtype={'filename': 'str'}` ?
Upon writing the comment it makes sense that it's the latter, but I want to make sure

Reply:
Yes,
Create this dict in the `__init__` method of the dataloader and then call it in the `dtypes` argument when we call the CSV reader.
Something like this:

 `pd.read_csv(...,dtypes=self.metadata_columns_type)`